app/src/main/java/ch/epfl/sdp/appart/AdActivity.java

```diff
@@ -84,15 +106,18 @@ private void updateTitle(String title) {
     }
 
     private void updatePhotos(List<String> references) {
-        LinearLayout horizontalLayout = findViewById(R.id.horizontal_children_Ad_linearLayout);
+        LinearLayout horizontalLayout =
+                findViewById(R.id.horizontal_children_Ad_linearLayout);
         horizontalLayout.removeAllViews();
 
         for (int i = 0; i < references.size(); i++) {
             String sep = FirebaseLayout.SEPARATOR;
-            String fullRef = FirebaseLayout.ADS_DIRECTORY + sep + adId + sep + references.get(i);
+            String fullRef =
+                    FirebaseLayout.ADS_DIRECTORY + sep + adId + sep + references.get(i);
```

Comment:
I know I'm like a religious man with its bible, bus there exists a builder for the path ! Maybe we will change it in the refactor since you only reformatted the line.

Reply:
You're right! I noticed the PathBuilder isn't used in a couple of places, but I lazily avoided modifying them, that's on me ^^'